reference/core/crds.md

```diff
@@ -37,7 +37,7 @@ Note that the `namespace` must be declared in the `metadata`, but if needed, `am
 
 ## Supported CRDs
 
-The full set of CRDs supported by the Ambassador Edge Stack:
+You can run `kubectl get ambassador-crds` to see the full set of CRDs supported by the Ambassador Edge Stack:
```

Comment:
This will actually get the CRs that you've created to configure Ambassador, rather the set of types (e.g. it'll show you all your `Mapping`s, all your `Host`s, etc., rather than the table of types below).

I'd just drop this change, and let the existing text stand.

Reply:
`kubectl api-resources | grep -F getambassador.io`?